client/package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
     "bootstrap": "^4.6.0",
+    "eslint": "^7.32.0",
```

Comment:
This isn't needed as we have eslint by default. You can uninstall this.

Reply:
fixed